Makefile

```diff
@@ -35,7 +35,7 @@ export GOCACHE=/tmp/cache
 test:
 	@echo ">> running unit tests (without cache)"
 	@rm -rf $(GOCACHE)
-	@go test -v -race -timeout=5m $(shell go list ./...);
+	@go test -v -race -timeout=10m $(shell go list ./...);
```

Comment:
I took the liberty to increase this to 10m since the pipeline timed out a few times.